pfs/lineLists/skyLines.txt

```diff
@@ -1,1769 +1,655 @@
-# lambda     Intensity  Species  Status
-# nm,vacuum
+# Columns:
+# 1: wavelength (nm)
+# 2: intensity (arbitrary units)
+# 3: description (ionic species)
+# 4: status (bitmask)
 #
-# Status:
-#  0   Isolated, good
-#  1   Not visible
-#  2   Blend; don't use
-#  4   Unknown; don't use
+# Status bitmask elements:
+# GOOD=0: Line is good
+# NOT_VISIBLE=1: Line is not typically visible in PFS (maybe too faint)
+# BLEND=2: Line is blended with other line(s)
+# SUSPECT=4: Line is of suspect quality
+# REJECTED=8: Line has been rejected from use in PFS
+# BROAD=16: Line is broader than normal
+# BAD=65535: Line is bad for any reason
```

Comment:
What is the meaning of 'bad' in this case? A non-zero status can be regarded as 'bad'. Or do you mean 'bad' for miscellaneous reasons? In which case isn't that another bitmask element?

Reply:
`BAD` is a catch-all bitmask: `0xFF`. It's not intended that individual lines be flagged as `BAD`, but having `BAD` makes it easy to specify a mask for rejecting lines. Maybe I should exclude it explicitly from the line list...